classes/person.rb

```diff
@@ -16,6 +18,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def name(name)
```

Comment:
- [ ] Kindly, make sure that you implement the method to add rentals. The person itself should be sent as a parameter to create the new rental, like this you'll achieve the 'has-many' association
```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)
  end
```